### PR TITLE
Add fixom costs to annuised investment costs

### LINKED ISSUE #136: Calculation of annuised investment cost does not include fix_om
The function is in scripts/prepare_scalars.py .

### PR DESCRIPTION
**Pull request by jnnr**
Fix #136

### CODE REVIEW

File: scripts/prepare_scalars.py [hunk 1]
```diff
@@ -78,7 +83,9 @@ def annuise_investment_cost(sc):
         invest_data["wacc"] = wacc
 
         annuised_investment_cost = invest_data.apply(
-            lambda x: annuity(x[var_name_cost], x["lifetime"], x["wacc"]), 1
+            lambda x: annuity(x[var_name_cost], x["lifetime"], x["wacc"])
+            + x[var_name_fixom_cost],
```
[the remark this answers is not in the document]
Comment (jnnr) [reply scripts/prepare_scalars.py:87]: No, as the unit reveals it is already given in relation to a year.

**Comment by jnnr** [comment]
@MaGering: Can you please make sure that the costs are annuised correctly? To do so, you can use the overview table created by `create_input_data_overview`, which shows the raw data, and compare it with the annuised costs that are saved in `results/_resources/scal_base.csv`

**Comment by MaGering** [comment]
> @MaGering: Can you please make sure that the costs are annuised correctly? To do so, you can use the overview table created by `create_input_data_overview`, which shows the raw data, and compare it with the annuised costs that are saved in `results/_resources/scal_base.csv`

I've checked and the results of `create_input_data_overview` contain values from `oemof-B3/raw/scalars_base.csv` instead of `oemof-B3/results/_resources/scal_base.csv` as indicated.

**Comment by jnnr** [comment]
> I've checked and the results of `create_input_data_overview` contain values from `oemof-B3/raw/scalars_base.csv` instead of `oemof-B3/results/_resources/scal_base.csv` as indicated.

That is what is expected. I cannot see where it is indicated otherwise?

**Comment by MaGering** [comment]
> That is what is expected. I cannot see where it is indicated otherwise?

I thought you indicated it. :-D  Then it must have been a misunderstanding. If I have now understood you correctly, then I should check the results of the calculation [here](https://github.com/rl-institut/oemof-B3/blob/1834e5519f51c362f64614ae8021d0e474cf256b/scripts/prepare_scalars.py#L80-L82) and find out whether these results are stored in `oemof-B3/results/_resources/scal_base.csv`?

This I have checked and it is the case.

**Pull request merged**